### PR TITLE
Update Pulumi Newsroom with new entries and logos

### LINKED ISSUE #1474: Pulumi Newsroom is missing coverage
### Problem description

Pulumi Newsroom is missing coverage as the last update was in October 2018. For easy upload, a coverage scan from October 2018-July 2019 is in the Google Doc here.

https://docs.google.com/spreadsheets/d/1CtZdFU2I30PF3_e4-snoaLbKPK8TsplQObkaDjSwQdo/edit#gid=0

### PR DESCRIPTION
**Pull request by bermudezmt**
Signed-off-by: bermudezmt <bermudez.mt@gmail.com>

### Related issues (optional)

Closes #1474

### CODE REVIEW

File: data/newsroom.toml [hunk 1]
```diff
@@ -23,22 +23,341 @@ summary = "Company funded by Madrona Venture Group and Tola Capital to help comp
...
+date = "15th October 2018"
+title = "GitHub introduces GitHub Actions to further automate the art of software development"
+img = "/logos/press/geekwire.png"
+url = "https://www.geekwire.com/2018/github-introduces-github-actions-automate-art-software-development/"
+
+[[coverage]]
+date = "15th October 2018"
 title = "Pulumi Bridges The Gap Between Cloud-Native Development And Infrastructure As Code"
 img = "/logos/press/forbes.png"
-url = "https://www.forbes.com/sites/janakirammsv/2018/10/16/pulumi-bridges-the-gap-between-cloud-native-development-and-infrastructure-as-code/"
```
Comment (ericrudder) [inline data/newsroom.toml]: intentional?

**Comment by justinvp** [comment]
> i don't quite understand the preview (last 5) vs. long term stuff

Yeah, this adds tons of "Recent News" links to the `/about` page now. How about we limit the `/about` page's press release and news items to the most recent 5 (as Eric suggests), and then (as a separate change; next milestone?) consider adding new `/about/press-releases` and `/about/news` pages that shows all press releases and news links, that we'd link to underneath the  most recent items for each?

To limit these to just the first 5 items, you can modify `layouts/page/about.html`:

Line 102:

```diff
- {{ range $.Site.Data.newsroom.releases }}
+ {{ range first 5 $.Site.Data.newsroom.releases }}
```

Line 114:

```diff
- {{ range $.Site.Data.newsroom.coverage }}
+ {{ range first 5 $.Site.Data.newsroom.coverage }}
```

**Comment by bermudezmt** [comment]
@justinvp It does look better with just the 5. I added an issue for the new pages here](https://github.com/pulumi/docs/issues/1481).

File: data/newsroom.toml (hunk 1 above)
Comment (bermudezmt) [reply data/newsroom.toml]: I think it's some kind of tracking info. When you take it out, a new one gets generated.

**Pull request merged**